src/shell/lisa_shell

```diff
@@ -527,6 +527,14 @@ $WLTEST_HOME/test_series "$@"
 deactivate
 }
 
+################################################################################
+# LISA Shell buildroot rootfs generation functions
+################################################################################
+
+function lisa-gen-buildroot-rootfs {
+	$LISA_HOME/src/shell/lisa_gen_buildroot_rootfs $@
```

Comment:
double quotes around "$@" are needed
https://stackoverflow.com/questions/37141039/is-there-any-difference-between-and